src/Azure.Functions.Cli/StaticResources/Dockerfile.dotnet9Isolated

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0-preview-noble-amd64 AS installer-env
+
+COPY . /src/dotnet-function-app
+RUN cd /src/dotnet-function-app && \
+mkdir -p /home/site/wwwroot && \
+dotnet nuget add source https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsTempStaging/nuget/v3/index.json && \
+dotnet publish *.csproj --output /home/site/wwwroot
+
+# To enable ssh & remote debugging on app service change the base image to the one below
+# FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4-dotnet-isolated9.0-appservice
+# TODO: change this back to the actual base image when it's released
+FROM mcr.microsoft.com/azure-functions/dotnet-isolated:4.35.1.2-dotnet-isolated9.0-appservice
```

Comment:
This should not be leveraging a pinned host version tag.

Reply:
changed to published official version